### ISSUE TITLE
current popovers hide the bottom of the container

### ISSUE THREAD

**Issue by peterstadler** [Edirom/WeGA-WebApp#174]
thus hiding the identifier, see "Ebers" in A060147

**Comment by peterstadler** [comment]
still an issue when the title breaks stretches over more than one line. see "Gutschmids" in A045245

**Comment by peterstadler** [comment]
it does work for "Deine beiden Briefe No 18 u 19" in A042755. Problem seems to be when the titles of the inner items have different length.

**Comment by jakosch** [comment]
Als Workaround padding/margin-bottom von `.popover .popover-content .item-content > div.media-body` erhöhen? 

![shot](https://user-images.githubusercontent.com/4886121/51077307-0d4bd800-16a5-11e9-95f4-651644399c5a.png)

**Comment by peterstadler** [comment]
Ja, gute Idee! Habe gerade nachgeschaut und `1.5 em` scheint mir ein guter Wert, da das die Schriftgröße des `h3.item-title` ist (welcher mit einer Extrazeile den Versatz des `div.item-content` erzeugt).
Möchtest Du @jakosch ?

**Comment by jakosch** [comment]
`1.5em` führt dazu, dass bei vielen popovers doch auffällig viel Whitespace unten entsteht und ggf. sogar scrolling ausgelöst wird, wo es gar nicht nötig ist. Minimalinvasiv wären so `4px`.

**Comment by jakosch** [comment]
Es geht doch nur um das Problem der abgeschnittenen Schrift - das braucht gar nicht viel Platz?

**Comment by peterstadler** [comment]
Bei mir in Firefox fehlt aber eine ganze Menge, siehe Screenshot 
![bildschirmfoto 2019-01-13 um 22 06 58](https://user-images.githubusercontent.com/2247936/51090666-a2bb9a80-177f-11e9-8585-7acf73aa9027.png)

**Comment by peterstadler** [comment]
another one to try: "Brief von Dresden und Gänsbacher" in A064157

**Comment by peterstadler** [comment]
deferred until we updated to Bootstrap4

**Comment by peterstadler** [comment]
Issue still exists (A064157): 
<img width="510" alt="Bildschirmfoto 2023-09-20 um 09 47 03" src="https://github.com/Edirom/WeGA-WebApp/assets/2247936/31393238-8bfd-4c3a-af69-d2fa7198914a">
